### DISCUSSION TITLE
doublet rate after demultiplexing by cellranger

### DISCUSSION THREAD

**Discussion by d-lashgari** [plger/scDblFinder#112] [Q&A]
Hi Dev Team,

Thank you for the tool; it's very useful. I have a question regarding the doublet rate. I am working with a dataset that includes multiple samples, each consisting of 7 different conditions multiplexed by 7 hashtags. The dataset has been processed with CellRanger and is already demultiplexed. However, I suspect there are doublets that were not detected by the demultiplexing process, specifically those that share the same hashtags.

For each sample, I have merged the demultiplexed conditions, as these were originally in the same well together, and I plan to run scdblfinder on this merged data. I believe I need to adjust the doublet rate, assuming that after demultiplexing, only 1/7 of the doublets remain. I calculate the adjusted doublet rate as 0.00143 * ncol(data) / 1000, based on the 0.01 rate per 1000 cells, divided by 7.

Could you confirm if this approach is correct?

All the best

**Comment by plger** [comment]
Hi,
I think your reasoning is correct, the only possible issue is that the number of cells relevant for the calculation of the effective rate depends on what was input into a single capture, rather than the merged one. Assuming that your samples are relatively homogeneous for the 7 hashtags, you should be able to use the average number of cells per capture as the basis for your calculations.
Given the uncertainty, you can also increase the `dbr.sd` argument, which gives the uncertainty of the `dbr` (by default it's 40% of the `dbr`), and will result in more weight being given to the optimization of the classification accuracy vs the expected number of doublets.
Best,
Pierre-Luc
